R-packages/evalcast/R/get_predictions.R

```diff
@@ -29,6 +29,8 @@
 #' @param signal_aggregation_dt for any data format, 
 #'   [covidcast::aggregate_signals()] can perform leading and lagging for you.
 #'   See that documentation for more details.
+#' @param use_covidcast this argument indicates whether your forecaster uses
+#'   covidcast data.  True by default.
```

Comment:
We should carefully explain  what will happen if you don't. Specifically, it looks like the forecaster needs to have its first argument be a NULL data frame.

Reply:
I'm actually not sure what we should expect to happen if we don't use covidcast data. Then we're just running their forecaster on their data and formatting it to work with this package's other tools? Maybe we should rethink: if they don't want any data, call a different function called `make_compatible_predictions()` or something.

Reply:
+1.  Is there a specific use case in mind for this argument we have here?

Reply:
I mean the idea was to use other forecasters.  Do we want to completely seperate that out from get_predictions?

Reply:
Maybe I didn't fully think this through before. It seems like there are two cases:

1. Someone has a forecaster that they want to evaluate with evalcast but using no covidcast data.
2. Someone wants some data from covidcast, but also some other data.

I think for the first case, we should write a separate function that somehow makes a prediction card that conforms. It's not really clear how this would work. Maybe something that allows one to "fill up" a prediction card with whatever they want?

For the second case, the additional data can just be interior to the forecaster, so this is maybe not an issue. But I think it *is* worth providing a demonstration. Suppose I want to do an AR(1) for weekly cases and also include unemployment claims. How would you do that?